src/en/infernalvoidscans/src/eu/kanade/tachiyomi/extension/en/infernalvoidscans/InfernalVoidScans.kt

```diff
@@ -1,14 +1,16 @@
 package eu.kanade.tachiyomi.extension.en.infernalvoidscans
 
-import eu.kanade.tachiyomi.multisrc.mangathemesia.MangaThemesia
-import eu.kanade.tachiyomi.network.GET
-import eu.kanade.tachiyomi.source.model.FilterList
-import okhttp3.Request
+import eu.kanade.tachiyomi.multisrc.iken.Iken
+import eu.kanade.tachiyomi.source.model.MangasPage
+import eu.kanade.tachiyomi.util.asJsoup
+import kotlinx.serialization.json.Json
+import okhttp3.Response
+import uy.kohesive.injekt.injectLazy
 
-class InfernalVoidScans : MangaThemesia(
+class InfernalVoidScans : Iken(
     "Infernal Void Scans",
```

Comment:
while you are at it, change the name to "Hive Toon" or similar here and in build.gradle file

Reply:
should i also change the path/folder names or should that stay the same?